Gravity/Gravity.Test/TestClasses/GravityLevel2.cs

```diff
@@ -9,5 +9,8 @@ public class GravityLevel2 : BaseDto
 	{
 		[RelativityObjectField("97E0BA08-9291-416A-9CE1-8D36EBDF9113", RdoFieldType.FixedLengthText, 255)]
 		public string Name { get; set; }
+
+		[RelativityObjectField("CE1E19FF-5603-4241-847E-72D2BD790984", RdoFieldType.SingleObject)]
+		public GravityLevel3 GravityLevel3SingleObj { get; set; }
```

Comment:
This new field (and type) should be reflected in an updated TestingApplication.xml. You can extract and use the one in the RAP.

Reply:
@Arithmomaniac Why do we need TestingApplication.xml if it is already stored in the RAP?

Reply:
Makes sense, I will replace the file.